test/suite/test_bug006.py

```diff
@@ -54,21 +54,17 @@ def test_bug006(self):
         # Table operations should fail, the cursor is open.
         self.assertRaises(
             wiredtiger.WiredTigerError, lambda: self.session.drop(uri, None))
-        self.assertRaises(
-            wiredtiger.WiredTigerError,
-            lambda: self.session.rename(uri, self.uri + "new", None))
         self.assertRaises(
             wiredtiger.WiredTigerError, lambda: self.session.salvage(uri, None))
         self.assertRaises(
             wiredtiger.WiredTigerError, lambda: self.session.upgrade(uri, None))
         self.assertRaises(
             wiredtiger.WiredTigerError, lambda: self.session.verify(uri, None))
 
+        self.session.checkpoint()
```

Comment:
Why was this needed?

Reply:
Without a checkpoint, the session salvage call returns an error "resource busy". This may be because we need to clear dirty data. I can alternatively change session.salvage to use the function` salvageUntilSuccess()` - which loops  calling salvage, checking if the error is EBUSY and checkpointing.

Reply:
Thanks for the explanation! I prefer the sound of `salvageUntilSuccess`, since it appears to map to the original code a little better (presumably there was a reason we called `renameUntilSuccess` - possibly this exact issue).

Reply:
Yup, I figured we needed checkpoint from renameUntilSuccess function - updated here [362d25d](https://github.com/wiredtiger/wiredtiger/pull/10282/commits/362d25dbd979b7aa8202bfd8a11eed3b42e3f2ba)